fdbserver/masterserver.actor.cpp

```diff
@@ -1233,6 +1241,15 @@ ACTOR Future<Void> serveLiveCommittedVersion(Reference<MasterData> self) {
 				reply.locked = self->databaseLocked;
 				reply.metadataVersion = self->proxyMetadataVersion;
 				reply.minKnownCommittedVersion = self->minKnownCommittedVersion;
+				std::size_t count = 0;
+				if (self->percentageOfTagsToBeSent > 100) {
+					count = rand() % (self->allTags.size() + 1);
```

Comment:
You should  use `deterministicRandom()` as simulation may break because it.

Reply:
Good point. Done.